pkg/discover/cloud_foundry/discover.go

```diff
@@ -10,77 +10,83 @@ import (
 
 type ValidationErrorList []error
 
-func Discover(cfApp AppManifest) (Application, error) {
-
-	timeout := 60
-	if cfApp.Timeout != 0 {
-		timeout = int(cfApp.Timeout)
-	}
-	services, err := marshalUnmarshal[Services](cfApp.Services)
-	if err != nil {
-		return Application{}, err
-	}
-	routeSpec := parseRouteSpec(cfApp.Routes, cfApp.RandomRoute, cfApp.NoRoute)
+func Discover(cfManifest CloudFoundryManifest) ([]Application, error) {
+	// TODO
+	// cfManifest.Space
+	// cfManifest.Version
```

Comment:
@jordigilh 
Regarding the `Space` I'll push other changes to check one particular space or to iterated among all of them.
Regarding the `Version` do you have any suggestion?

Reply:
For Spaces, I'd suggest we make it mandatory to pass the space name. I don't think it makes sense to let it scrape the whole platform, but we should ask the team for more information on this one.